content/en/docs/tutorials/hello-minikube.md

```diff
@@ -90,14 +90,17 @@ tutorial has only one Container. A Kubernetes
 Pod and restarts the Pod's Container if it terminates. Deployments are the
 recommended way to manage the creation and scaling of Pods.
 
-1. Use the `kubectl create` command to create a Deployment that manages a Pod. The
+1. Katacoda environment only: At the top of the terminal pane, click the plus sign, and then click open a new terminal.
+
+2. Use the `kubectl create` command to create a Deployment that manages a Pod. The
```

Comment:
Its good idea which will require less change for future modification too

Reply:
> Its good idea which will require less change for future modification too
Hi, I'm not feeling clear, it's mean I need to change `1.` to `*` ?

Reply:
> Its good idea which will require less change for future modification too

Its good idea which will require less change for future modification too
Hi, I'm not feeling clear, it's mean I need to change `1.` to `*` ?

Reply:
The recommended way to assign numbers is to always use `1.` instead of explicitly assign `1.`, `2.` etc.

Reply:
> The recommended way to assign numbers is to always use `1.` instead of explicitly assign `1.`, `2.` etc.

Okay. Does the website render support this way and render will convert it automatically?

Reply:
There is a markdown parser behind hugo, which is GoldMark I believe.
You can learn some [Markdown syntax on this](https://www.markdownguide.org/basic-syntax#ordered-lists).

Reply:
Thank for your reply.